cleo_sdk/CLEO.h

```diff
@@ -434,15 +481,21 @@ DWORD WINAPI CLEO_GetVarArgCount(CRunningScript* thread); // peek remaining var-
 extern SCRIPT_VAR* opcodeParams;
 extern SCRIPT_VAR* missionLocals;
 
+SCRIPT_VAR* WINAPI CLEO_GetParamsCollectiveArray(); // get pointer to 'SCRIPT_VAR[32] opcodeParams'. Used by Retrieve/Record opcode params functions
```

Comment:
maybe it should be called `CLEO_GetOpcodeParamsArray` ? `Collective` is a bit strange word here

Reply:
Done